src/pickley/cli.py

```diff
@@ -517,7 +546,7 @@ def cmd_list(border, format, verbose):
         python = runez.dim("-not needed-") if pspec.is_uv else manifest and manifest.python_executable
         delivery = runez.dim("-") if pspec.is_uv else manifest and manifest.delivery
         package_manager = manifest and manifest.package_manager
-        if package_manager and package_manager != default_package_manager:
+        if package_manager and package_manager != default_package_manager:  # pragma: no cover, uncommon
```

Comment:
I'm not a fan of this no-cover either, just take the not-100%-coverage or write a test

Reply:
It's a hard balance to strike, I'm struggling with it :)
I use code coverage to find what's the next edge case I'm missing a test for, and mark with `no cover` the ones I think are not worth setting up a complex test for.

Later, every now and then, I search for `no cover`, remove it, and see if I could add a test now (this is here to stay, and an even a complex test would be warranted)

Reply:
Also, after a refactor, I typically remove all `no cover` pragmas, comment out all the complex not-so-useful mocks, and see which part of the code is now ripe for removal entirely (ie: code that remained "covered" only because there is an esoteric test that exercised it)